src/themes/standard.ts

```diff
@@ -82,36 +95,62 @@ export const standardFlowbiteTheme: CustomFlowbiteTheme = {
-          "bg-transparent border-current text-red-600 enabled:hover:bg-red-100 focus:ring-red-400 dark:bg-red-900 dark:enabled:hover:bg-red-800 dark:focus:ring-red-700",
-        primary:
-          "bg-transparent border-current text-primary-700 enabled:hover:bg-primary-100 focus:ring-primary-300 dark:bg-primary-600 dark:enabled:hover:bg-primary-700 dark:focus:ring-primary-800",
-        info: "bg-transparent border-current text-blue-600 enabled:hover:bg-blue-100 focus:ring-blue-400 dark:bg-blue-900 dark:enabled:hover:bg-blue-800 dark:focus:ring-blue-700",
-        secondary:
-          "bg-transparent border-current text-gray-300 text-gray-900 focus:ring-gray-200 enabled:hover:bg-gray-100 dark:border-gray-600 dark:bg-gray-600 dark:text-white dark:focus:ring-gray-700 dark:enabled:hover:border-gray-700 dark:enabled:hover:bg-gray-700",
-        success:
-          "bg-transparent border-current text-green-600 focus:ring-green-300 enabled:hover:bg-green-100 dark:bg-green-600 dark:focus:ring-green-800 dark:enabled:hover:bg-green-700",
-        warning:
-          "bg-transparent border-current text-yellow-600 enabled:hover:bg-yellow-100 focus:ring-yellow-300 dark:bg-yellow-300 dark:enabled:hover:bg-yellow-600 dark:focus:ring-yellow-500",
+        failure: [
+          "text-red-600 bg-transparent border-current enabled:hover:bg-red-100 focus:ring-red-400",
+          "dark:text-red-400 dark:bg-transparent dark:enabled:hover:text-white dark:enabled:hover:bg-red-800 dark:focus:ring-red-700",
+        ].join(" "),
+        primary: [
+          "text-primary-700 bg-transparent border-current enabled:hover:bg-primary-100 focus:ring-primary-300",
+          "dark:text-primary-400 dark:bg-transparent dark:enabled:hover:text-white dark:enabled:hover:bg-primary-700 dark:focus:ring-primary-800",
+        ].join(" "),
+        info: [
+          "text-blue-600 bg-transparent border-current enabled:hover:bg-blue-100 focus:ring-blue-400",
+          "dark:text-blue-400 dark:bg-transparent dark:enabled:hover:text-white dark:enabled:hover:bg-blue-800 dark:focus:ring-blue-700",
+        ].join(" "),
+        secondary: [
+          "text-secondary-700 bg-transparent border-current enabled:hover:bg-secondary-100 focus:ring-secondary-300",
+          "dark:text-secondary-300 dark:bg-transparent dark:enabled:hover:text-white dark:enabled:hover:bg-secondary-700 dark:focus:ring-secondary-800",
+        ].join(" "),
+        success: [
+          "text-green-600 bg-transparent border-current focus:ring-green-300 enabled:hover:bg-green-100",
+          "dark:text-green-400 dark:bg-transparent dark:enabled:hover:text-white dark:focus:ring-green-800 dark:enabled:hover:bg-green-700",
+        ].join(" "),
+        warning: [
+          "text-yellow-600 bg-transparent border-current enabled:hover:bg-yellow-100 focus:ring-yellow-300",
+          "text-yellow-400 dark:bg-transparent dark:enabled:hover:text-white dark:enabled:hover:bg-yellow-600 dark:focus:ring-yellow-500",
```

Comment:
This uses both yellow 600 and 400, should one be prepended by `dark:`

Reply:
Good eye. Thank you.